### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by bc-svc-local**
This pull request was created automatically. Please check new translations, approve and deliver them.

### CODE REVIEW

**Comment by BC-krasnoshapka** [comment]
Full translations onboarding will be done in separate PR.

File: core/messages/de.json [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+        "loading": "Wird geladen …",
+        "carouselTitle": "Verwandte Produkte",
+        "noGalleryText": "In Kürze verfügbar",
+        "DescriptionAndReviews": {
+            "description": "Beschreibung",
+            "warranty": "Garantie",
+            "loading": "Wird geladen …",
+            "reviews": "Bewertungen",
+            "reviewsCount": "Count:",
+            "unreviewed": "This product hasn't been reviewed yet.",
```
Comment (bc-alexsaiannyi) [inline core/messages/de.json:37]: I see that we miss some translations across phrases like here. Is it expected?

Comment (BC-krasnoshapka) [reply core/messages/de.json:37]: Yes, not all phrases are translated already, more will come in future updates.

**Comment by BC-krasnoshapka** [comment]
Successfully tested locally using `Accept-Language` header (temporary multi-lang implementation).

Danish, da
Screenshot 2024-05-16 at 14 23 02

French, fr
Screenshot 2024-05-16 at 14 22 12

Spanish Mexican, es-MX
Screenshot 2024-05-16 at 14 20 20

**Review by BC-krasnoshapka — APPROVED** [review]

**Pull request merged**